app/utils/units.js

```diff
@@ -21,4 +21,17 @@ var defaultLocation = {
   mapType: 'road'
 };
 
-export {imageTypes, genders, searchGenders, defaultLocation};
+var dobOptions = [
+  {label: '0-3 years', value: 1.5},
+  {label: '3-6 years', value: 4.5},
+  {label: '6-9 years', value: 7.5},
+  {label: '9-12 years', value: 10.5},
+  {label: '12-15 years', value: 13.5},
+  {label: '15-20 years', value: 17.5},
+  {label: '20-25 years', value: 22.5},
+  {label: '25+ years', value: 27.5}
+];
```

Comment:
@redyaffle  probably need separation options for search here like
`var SearchDobOptions = [{label: 'all', value: null}].concat(dobOptions)`

Notice we do similar thing for gender and searchGender.